docs/WebsocketAPIV4.md

```diff
@@ -0,0 +1,235 @@
+## WebSocket protocol v4 details
+
+WebSocket Protocol v4 requires an Exasol version of at least 8.31.0. It follows the RFC 6455 document of the IETF.
```

Comment:
If DB Exa to Exa changes are merged this week, then DB version will be 8.32.0.

Reply:
Done